lib/src/rating/svgStarHollow.svg.html

```diff
@@ -0,0 +1,31 @@
+<svg
+    [attr.width]="width"
+    [attr.height]="height"
+    viewBox="0 0 24 24"
+    xmlns="http://www.w3.org/2000/svg"
+    xmlns:xlink="http://www.w3.org/1999/xlink"
+>
+    <title *ngIf="title">{{ title }}</title>
+    <defs>
+        <path
+            id="path-1-unselected"
+            d="m10.29796,13.734l4.71501,3.2844l-1.90297,-5.3504c-0.067,-0.207 0.007,-0.432 0.183,-0.56l4.72728,-3.108l-5.82128,0c-0.217,0 -0.408,-0.376 -0.475,-0.582l-1.72455,-5.41103l-1.80945,5.41203c-0.067,0.205 -0.259,0.581 -0.476,0.581l-5.714,0l4.621,3.108c0.176,0.128 0.25,0.353 0.183,0.56l-1.7939,5.3504l4.70186,-3.2844c0.087,-0.064 0.19,-0.095 0.293,-0.095c0.103,0 0.206,0.031 0.293,0.095zm9.70204,-6.43564c-0.1,-0.2 -0.3,-0.3 -0.5,-0.3l-6.9,0l-2.1,-6.59836c-0.1,-0.3 -0.3,-0.4 -0.5,-0.4c-0.2,0 -0.4,0.1 -0.5,0.3l-2.1,6.69836l-6.9,0c-0.2,0 -0.4,0.1 -0.5,0.3c0,0.2 0,0.5 0.2,0.6l5.6,3.90164l-2.1,6.5c-0.1,0.2 -0.06934,0.35791 0.13066,0.55791c0.1,0.1 0.36934,0.14209 0.56934,0.04209l5.6,-4l5.6,4c0.1,0.1 0.5,0.1 0.6,0c0.2,-0.1 0.2,-0.4 0.2,-0.6l-2.1,-6.5l5.6,-3.90164c0.1,-0.1 0.1,-0.4 0.1,-0.6z"
+        />
+    </defs>
+    <g>
+        <rect fill="none" id="canvas_background" height="6.61537" width="6.61537" y="-1" x="-1" />
+    </g>
+    <g>
+        <title *ngIf="title">{{title}}</title>
+        <g fill-rule="evenodd" fill="none" id="Page-1">
+            <g id="Icons-main-navigation/Star-unselected/Darkblue">
+                <g id="Icon">
+                    <mask transform="translate(2,2) " id="mask-unselected">
```

Comment:
I think we should remove all IDs since we are rendering at least 5 instances of either of the SVGs.